### ISSUE TITLE
Fix loss function

### ISSUE THREAD

**Issue by toml01** [BioCellColman/BioCell#7]
Loss function as of now is `binary_crossentropy`

**Comment by toml01** [comment]
changed it to `categorical_crossentropy`

**Comment by toml01** [comment]
had to change last layer's activation to `softmax` too.
